instructions/index.html

```diff
@@ -1,11 +1,34 @@
-Vanessa Rodriguez
-About: 
-    Current registered nurse with a passion for technology and healthcare. Currently working on my dream of becoming a software developer so that I can offer powerful solutions to patients and healthcare professionals.
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>About Me</title>
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        Vanessa Rodriguez
+        <section>
+            
+            <h1>About</h1> 
+                <p>Current registered nurse with a passion for technology and healthcare.
+                    Currently working on my dream of becoming a software developer so that I can offer 
+                    powerful solutions to patients and healthcare professionals.</p>
```

Comment:
Even though it intuitively seems like the `<p>` would be "under" the `<h1>`, they are actually sibling elements (at the same level of nesting) so they should be indented the same amount:

```suggestion
            <h1>About</h1> 
            <p>Current registered nurse...</p>
```
Same applies for the other `<h1>`'s below